Source/src/modules/ModuleCamera.cpp

```diff
@@ -135,20 +140,20 @@ void Hachiko::ModuleCamera::RemoveCameraComponent(ComponentCamera* camera)
     camera_buffer.erase(std::remove(camera_buffer.begin(), camera_buffer.end(), camera), camera_buffer.end());
 }
 
-void Hachiko::ModuleCamera::SetMainCamera(ComponentCamera* camera)
+void Hachiko::ModuleCamera::SetRenderingCamera(ComponentCamera* camera)
 {
-    main_camera = static_cast<ComponentCamera*>(camera);
-    main_camera_game_object = camera->GetGameObject();
+    rendering_camera = static_cast<ComponentCamera*>(camera);
```

Comment:
why do we make this cast here?

Reply:
To be honest, no idea, probably because it was used with `GetComponent(Component::Type::CAMERA)` instead of `GetComponent<ComponentCamera>()`. I haven't touched older code, but in the near future I will be doing some changes to the ModuleCamera so we can address those that time. 👍

Reply:
yeah, I saw you didn't change anything else but the variable name, but I found it curious.